themes/src/main/resources-community/theme/base/login/messages/messages_es.properties

```diff
@@ -68,10 +68,23 @@ country=Pa\u00EDs
 emailVerified=Email verificado
 gssDelegationCredential=GSS Delegation Credential
 
+loginTotpIntro=Es necesario configurar un generador de claves de un sólo uso para acceder a esta cuenta
 loginTotpStep1=Instala <a href=\"https://freeotp.github.io/\" target=\"_blank\">FreeOTP</a> o Google Authenticator en tu tel\u00E9fono m\u00F3vil. Ambas aplicaciones est\u00E1n disponibles en <a href=\"https://play.google.com\">Google Play</a> y en la App Store de Apple.
 loginTotpStep2=Abre la aplicaci\u00F3n y escanea el c\u00F3digo o introduce la clave.
 loginTotpStep3=Introduce el c\u00F3digo \u00FAnico que te muestra la aplicaci\u00F3n de autenticaci\u00F3n y haz clic en Enviar para finalizar la configuraci\u00F3n
+loginTotpStep3DeviceName=Introduce un nombre de dispositivo para que te ayude a gestionar tus dispositivos OTP.
```

Comment:
```suggestion
loginTotpStep3DeviceName=Introduce un nombre del dispositivo para que te ayude a gestionar tus dispositivos OTP.
```

Reply:
I don't think this translation is correct